homeassistant/components/entur_public_transport/sensor.py

```diff
@@ -0,0 +1,249 @@
+"""
+Real-time information about public transport departures in Norway.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.entur_public_transport/
+"""
+from datetime import datetime, timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME,
+    CONF_SHOW_ON_MAP)
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.util.dt as dt_util
+
+REQUIREMENTS = ['enturclient==0.2.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+API_CLIENT_NAME = 'homeassistant-homeassistant'
+
+ATTRIBUTION = "Data provided by entur.org under NLOD"
+
+CONF_STOP_IDS = 'stop_ids'
+CONF_EXPAND_PLATFORMS = 'expand_platforms'
+CONF_WHITELIST_LINES = 'line_whitelist'
+CONF_OMIT_NON_BOARDING = 'omit_non_boarding'
+CONF_NUMBER_OF_DEPARTURES = 'number_of_departures'
+
+DEFAULT_NAME = 'Entur'
+DEFAULT_ICON_KEY = 'bus'
+
+ICONS = {
+    'air': 'mdi:airplane',
+    'bus': 'mdi:bus',
+    'metro': 'mdi:subway',
+    'rail': 'mdi:train',
+    'tram': 'mdi:tram',
+    'water': 'mdi:ferry',
+}
+
+SCAN_INTERVAL = timedelta(seconds=45)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STOP_IDS): vol.All(cv.ensure_list, [cv.string]),
+    vol.Optional(CONF_EXPAND_PLATFORMS, default=True): cv.boolean,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_SHOW_ON_MAP, default=False): cv.boolean,
+    vol.Optional(CONF_WHITELIST_LINES, default=[]): cv.ensure_list,
+    vol.Optional(CONF_OMIT_NON_BOARDING, default=True): cv.boolean,
+    vol.Optional(CONF_NUMBER_OF_DEPARTURES, default=2):
+        vol.All(cv.positive_int, vol.Range(min=2, max=10))
+})
+
+
+ATTR_STOP_ID = 'stop_id'
+
+ATTR_ROUTE = 'route'
+ATTR_ROUTE_ID = 'route_id'
+ATTR_EXPECTED_AT = 'due_at'
+ATTR_DELAY = 'delay'
+ATTR_REALTIME = 'real_time'
+
+ATTR_NEXT_UP_IN = 'next_due_in'
+ATTR_NEXT_UP_ROUTE = 'next_route'
+ATTR_NEXT_UP_ROUTE_ID = 'next_route_id'
+ATTR_NEXT_UP_AT = 'next_due_at'
+ATTR_NEXT_UP_DELAY = 'next_delay'
+ATTR_NEXT_UP_REALTIME = 'next_real_time'
+
+ATTR_TRANSPORT_MODE = 'transport_mode'
+
+
+def due_in_minutes(timestamp: datetime) -> int:
+    """Get the time in minutes from a timestamp.
+
+    The timestamp should be in the format
+    year-month-yearThour:minute:second+timezone
```

Comment:
Is this comment still valid?

Reply:
No, and I thought I had updated that one, but obviously only in my mind..